CMakeLists.txt

```diff
@@ -36,7 +36,9 @@ set(BICYCLE_SOURCE
 set(BICYCLE_INCLUDE_DIR ${BICYCLE_SOURCE_DIR} ${BICYCLE_SOURCE_DIR}/../inc)
 add_definitions("-DBICYCLE_USE_DOUBLE_PRECISION_REAL=false")
 set_property(SOURCE ${BICYCLE_SOURCE} APPEND_STRING PROPERTY COMPILE_FLAGS
-    " -fno-math-errno -fassociative-math -freciprocal-math -fno-signed-zeros -fcx-limited-range")
+    " -fno-math-errno -fassociative-math -freciprocal-math -fcx-limited-range")
+#set_property(SOURCE ${BICYCLE_SOURCE} APPEND_STRING PROPERTY COMPILE_FLAGS
```

Comment:
I'm planning on cleaning up this PR and removing most of the commented code. Anything that's commented should have some note next to it.